controllers/homepage-routes.js

```diff
@@ -32,6 +32,15 @@ router.get("/", (req, res) => {
     .then(async (dbRecipeData) => {
       // pass a single recipe object into the homepage template
       const recipes = dbRecipeData.map((recipe) => recipe.get({ plain: true }));
+      recipes.sort((recipeOne, recipeTwo) => {
+        if (recipeOne.vote_count < recipeTwo.vote_count) {
+          return 1
+        }
+        if (recipeOne.vote_count > recipeTwo.vote_count) {
+          return -1
+        }
+        return 0
+      })
```

Comment:
this change was in main, but not develop so I am merging it in to keep develop up to date